### DISCUSSION TITLE
获取id

### DISCUSSION THREAD

**Discussion by naozhong1564856979** [simple-robot/Discussions#156] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

QQ频道

### 疑问描述

快速开始那里有一个 `@Listener` 和 `@Filter` 通过输入一定的条件来让机器人发送信息，然后，用户发信息给机器人的时候，机器人接收到的频道id，子频道id还有用户id什么的如何拿出来

**Comment by ForteScarlet** [comment, marked as answer]
id 就在各个对象中。以子频道id为例，监听子频道消息事件的时候：
```java
ID id = event.getChannel().getId();
```

**Reply by naozhong1564856979** [reply]
ok, 我试试

**Reply by JacianZhu** [reply]
@naozhong1564856979  你这边是怎么登录的 robot 配置文件能贴出来吗
